code/modules/organs/robolimbs.dm

```diff
@@ -47,22 +47,24 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 	var/icon = 'icons/mob/human_races/robotic.dmi'       // Icon base to draw from.
 	var/monitor_icon = 'icons/mob/monitor_icons.dmi'     // Where it draws the monitor icon from.
 	var/unavailable_at_chargen                           // If set, not available at chargen.
-	var/unavailable_to_build							 // If set, can't be constructed.
-	var/lifelike										 // If set, appears organic.
-	var/skin_tone										 // If set, applies skin tone rather than part color Overrides color.
-	var/skin_color										 // If set, applies skin color rather than part color.
-	var/blood_color = "#030303"
-	var/blood_name = "oil"
+	var/unavailable_to_build                             // If set, can't be constructed.
```

Comment:
These changes are entirely whitespace normalization other than adding the new `modular_bodyparts` bool.